tests/e2e/unauthenticated_user_test/unathenticated_user_test.go

```diff
@@ -0,0 +1,84 @@
+// Unathenticated user tries to access endpoints
+
+package tests
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/HackIllinois/api/common/configloader"
+	event_models "github.com/HackIllinois/api/services/event/models"
+	mail_models "github.com/HackIllinois/api/services/mail/models"
+	notification_models "github.com/HackIllinois/api/services/notifications/models"
+	"github.com/HackIllinois/api/tests/common"
+	"github.com/dghubble/sling"
+	"gopkg.in/mgo.v2"
+)
+
+var attendee_client *sling.Sling
+var session *mgo.Session
+
+func TestMain(m *testing.M) {
+
+	cfg, err := configloader.Load(os.Getenv("HI_CONFIG"))
+
+	if err != nil {
+		fmt.Printf("ERROR: %v\n", err)
+		os.Exit(1)
+	}
+
+	attendee_client = sling.New().Base("http://localhost:8000").Client(nil).Add("Authorization", "FAKE_TOKEN")
```

Comment:
Can consider using the `common.GetSlingClient()` here, which generates a token with the specified role.

Example [here](https://github.com/HackIllinois/api/blob/staging/tests/e2e/unauthorized_actions_test/unauthorized_actions_test.go#L28)

Reply:
Actually, I intentionally did not use it since GetSlingClient would generate a token for the client (essentially making them authenticated), which is the opposite of what we are trying to test.

Reply:
Right, that makes sense